DuckDuckGo.xcodeproj/project.pbxproj

```diff
@@ -6359,7 +6359,7 @@
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = 0;
+				CURRENT_PROJECT_VERSION = $CURRENT_PROJECT_VERSION;
```

Comment:
We'll no longer set this manually.  I'm doing maintenance this week so I can update the release steps before doing the release myself.